### DISCUSSION TITLE
Solarized Light text contrast

### DISCUSSION THREAD

**Discussion by MobileAZN** [elementary/code#1116] [Ideas]
Can we get a little more contrast in the text color in the Solarized Light theme?  I like the colors, but that light grey is hard to read against the nice butter creme background.  I think black text would be a little much, but maybe a little more like a German Grey or something?  

I'm sure I could probably dig into the code, but I'm not setup yet to do the change and such and in the middle of a few other projects.

**Comment by misobadev** [comment]
Hope they can add theme support soon
